tests/unittest_builder.py

```diff
@@ -38,11 +38,12 @@
 import tempfile
 import textwrap
 import unittest
+import unittest.mock
```

Comment:
Probably missing something, but is this being used?

Reply:
It depends, running the whole test suite works. However `pytest tests/unittest_builder.py` currently fails. Adding that line fixes it. _Another indicator is that VS Code intellisense only works with it._
```
AttributeError: module 'unittest' has no attribute 'mock'
```
https://github.com/PyCQA/astroid/blob/29dcc35ac14ac21fe97502ac890a1faac936054e/tests/unittest_builder.py#L1020

Reply:
Ah, unrelated to this PR then. I was wondering why I didn't see `mock` anywhere in the diff.
Fine with me to keep this in!